### ISSUE TITLE
[photospline] Modern C++ Issues (Trac #1831)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1823]
A few of the bots complain about linking photospline after the cpp11 switch was thrown.

http://builds.icecube.wisc.edu/builders/OS%20X%20El%20Capitan/builds/851/steps/compile/logs/stdio

Migrated from https://code.icecube.wisc.edu/ticket/1831
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:38", 
    "description": "A few of the bots complain about linking photospline after the cpp11 switch was thrown.\n\nhttp://builds.icecube.wisc.edu/builders/OS%20X%20El%20Capitan/builds/851/steps/compile/logs/stdio", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067158057333", 
    "component": "combo reconstruction", 
    "summary": "[photospline] Modern C++ Issues", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-08-19T18:31:51", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160819T20:52:23: jvansanten commented:

> Can I assume that suitesparse was linked against libstdc++? It looks like spglam is either being linked against libc++ or not any standard library at all.

**Comment by icecube-trac** [comment]
Trac update at 20160819T21:58:32: nega commented:

> i'll try and take a poke at it this weekend. if you need access to the bot, slack me

**Comment by icecube-trac** [comment]
Trac update at 20160830T18:37:58: 
* jvansanten changed resolution from "" to "fixed"
* jvansanten commented:

> In changeset 149671:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="149671"
> Explicitly link spglam to libc++ on El Capitan. Closes #1823.
> ```


* jvansanten changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:38: 
* olivas changed milestone from "August 2016" to ""
* olivas commented:

> Milestone August 2016 deleted
